pkg/operator/ceph/csi/spec.go

```diff
@@ -235,6 +238,11 @@ const (
 	defaultLogLevel        uint8 = 0
 	defaultSidecarLogLevel uint8 = 0
 
+	// default leader election flags
+	defaultLeaderElectionLeaseDuration = 137 * time.Second
```

Comment:
how these times are justified?

Reply:
@parth-gr I asked the same question in the resolved comment https://github.com/rook/rook/pull/13573#discussion_r1453319924 :Xd

Reply:
I see they were the defaults before